Fitness/Controllers/GymDetailViewController.swift

```diff
@@ -141,35 +152,41 @@ class GymDetailViewController: UIViewController, UITableViewDelegate, UITableVie
         popularTimesTitleLabel.text = "POPULAR TIMES"
         contentView.addSubview(popularTimesTitleLabel)
         
+        days = []
+        days.insert(.sunday, at: 0)
+        days.insert(.monday, at: 1)
+        days.insert(.tuesday, at: 2)
+        days.insert(.wednesday, at: 3)
+        days.insert(.thursday, at: 4)
+        days.insert(.friday, at: 5)
+        days.insert(.saturday, at: 6)
+        
         var data: [Int]
         let date = Date()
         let todaysHours: DailyGymHours
         
-        switch date.getIntegerDayOfWeekToday() {
-        case 0:
+        switch days[date.getIntegerDayOfWeekToday() ?? 0] {
+        case .sunday:
             data = gym.popularTimesList.sunday
             todaysHours = gym.gymHours.zero ?? DailyGymHours(id: -1, dayOfWeek: 0, openTime: "", closeTime: "")
```

Comment:
why is `gym.gymHours.zero` an optional? Can we make it a non-optional to get rid of all the `??`

Reply:
It was initially an optional to work around the fact that backend didn't provide hours for days that gyms weren't open (I realize now that backend should have handled this on their end), but with the new use of GraphQL this shouldn't be an issue, and if it is I'll talk with backend and have them make the changes on their end.